cfbs/validate.py

```diff
@@ -356,6 +357,11 @@ def validate_build_step(name, module, i, operation, args, strict=False):
         validate_build_step(
             name, module, i, "replace", [n, to_replace, version, filename], strict
         )
+    elif operation == "patch":
+        # TODO: perhaps check that the patch file exists here instead of in `apply_patch`?
+        # TODO: what kinds of paths should be legal?
+        # TODO: more validation?
```

Comment:
Will you create a follow-up ticket for this? Maybe you can link it here (using the ticket number).